scripts/blocks-utils.js

```diff
@@ -173,4 +267,9 @@ export {
   getEnvType,
   decorateQuickLinks,
   fetchData,
+  getOriginUrl,
```

Comment:
do we need to export all these ?

Reply:
it gives more flexiblity to the user, we can remove later if not needed